<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="A2MD20092" name="io.mosip.preregistration.tests.IntegrationScenarios" tests="46" failures="5" timestamp="02 Jul 2019 13:56:07 GMT" time="1106.781" errors="2">
  <testcase name="preReg_IntTst_createAppDiscardUploadDoc" time="6.914" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_createAppUploadBookUpdate" time="22.907" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_retrivePreRegistrationDataForCancelAppointment" time="20.229" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_getPreRegistrationDataForBookedAppointment" time="22.503" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_updatePreRegistrationDataForExpiredApplication" time="23.632" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <failure type="java.lang.AssertionError" message="Failed stabilized connection with preregdb expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed stabilized connection with preregdb expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at io.mosip.preregistration.dao.PreregistrationDAO.setDate(PreregistrationDAO.java:65)
	at io.mosip.preregistration.tests.IntegrationScenarios.updatePreRegistrationDataForExpiredApplication(IntegrationScenarios.java:525)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </failure>
  </testcase> <!-- preReg_IntTst_updatePreRegistrationDataForExpiredApplication -->
  <testcase name="preReg_IntTst_consumedMultiplePRID" time="86.685" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_cosumedExpiredAppointment" time="25.681" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <failure type="java.lang.AssertionError" message="Failed stabilized connection with preregdb expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed stabilized connection with preregdb expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at io.mosip.preregistration.dao.PreregistrationDAO.setDate(PreregistrationDAO.java:65)
	at io.mosip.preregistration.tests.IntegrationScenarios.cosumedExpiredAppointment(IntegrationScenarios.java:1073)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </failure>
  </testcase> <!-- preReg_IntTst_cosumedExpiredAppointment -->
  <testcase name="preReg_IntTst_retrivePreRegistrationDataAfterBookingAppointment" time="24.352" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_createAppUploadDocDeleteDocByDocId" time="5.690" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_multipleDocumentUploadDeleteDocByPreId" time="9.616" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_expiredBatchJobService" time="25.401" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <failure type="java.lang.AssertionError" message="Failed stabilized connection with preregdb expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed stabilized connection with preregdb expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at io.mosip.preregistration.dao.PreregistrationDAO.setDate(PreregistrationDAO.java:65)
	at io.mosip.preregistration.tests.IntegrationScenarios.expiredBatchJobService(IntegrationScenarios.java:1102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </failure>
  </testcase> <!-- preReg_IntTst_expiredBatchJobService -->
  <testcase name="preReg_IntTst_retrivePreRegDataConsumedApplication" time="17.172" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_updateDemographicDataAfterBookingAppointMent" time="26.219" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_createAppUpdateDemoGetData" time="5.168" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_cancelAppointmentFetchCenterDetails" time="36.701" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_retrivePreRegistrationDataAfterUploadingDemographicDetails" time="5.345" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_fetchMultipleApplicationCreatedByUser" time="34.894" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_consumedBatchJobService" time="61.608" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at io.mosip.preregistration.util.PreRegistartionDataBaseAccess.getDataBaseConnection(PreRegistartionDataBaseAccess.java:35)
	at io.mosip.preregistration.util.PreRegistartionDataBaseAccess.getConsumedStatus(PreRegistartionDataBaseAccess.java:59)
	at io.mosip.preregistration.dao.PreregistrationDAO.getConsumedStatus(PreregistrationDAO.java:89)
	at io.mosip.util.PreRegistrationLibrary.getConsumedStatus(PreRegistrationLibrary.java:348)
	at io.mosip.preregistration.tests.IntegrationScenarios.consumedBatchJobService(IntegrationScenarios.java:1131)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </error>
  </testcase> <!-- preReg_IntTst_consumedBatchJobService -->
  <testcase name="preReg_IntTst_retrivePreRegistrationDataOfDiscardedApplication" time="21.362" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_fetchDiscardedApplication" time="23.856" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_createAppUpdateDiscard" time="34.236" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_createAppUploadDocDeleteDocByPreRegId" time="7.498" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_bookMultipleAppointmentForSamePRID" time="40.631" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_consumedMultiplePRIDWithInvalidPRID" time="73.360" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_multipleDocumentUpload" time="10.292" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_fetchMultipleUserCreatedByUser" time="29.880" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_cosumedBookedAppointment" time="24.539" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at io.mosip.preregistration.util.PreRegistartionDataBaseAccess.getDataBaseConnection(PreRegistartionDataBaseAccess.java:35)
	at io.mosip.preregistration.util.PreRegistartionDataBaseAccess.getConsumedStatus(PreRegistartionDataBaseAccess.java:59)
	at io.mosip.preregistration.dao.PreregistrationDAO.getConsumedStatus(PreregistrationDAO.java:89)
	at io.mosip.util.PreRegistrationLibrary.getConsumedStatus(PreRegistrationLibrary.java:348)
	at io.mosip.preregistration.tests.IntegrationScenarios.cosumedBookedAppointment(IntegrationScenarios.java:1051)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </error>
  </testcase> <!-- preReg_IntTst_cosumedBookedAppointment -->
  <testcase name="preReg_IntTst_uploadMultipleDocsForSameCategory" time="9.215" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_dataOfDiscardedApplication" time="11.330" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_bookAppointmentForDiscardedApplication" time="21.394" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_fetchCanceledAppointmentCreatedByUser" time="43.292" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_fetchBookedAppointmentCreatedByUser" time="53.711" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_getStatusOfBookedAppointment" time="22.673" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_getPreRegistrationDataForDiscardedApplication" time="12.078" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_uploadDocumentForDiscardedApplication" time="5.182" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_getPreRegistrationDataOfDiscardedApplication" time="6.627" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_delDocByDocIdForDiscardedApplication" time="3.545" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_getPreRegistrationDataForPendingApplication" time="6.548" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_bookAppForInvalidDate" time="15.765" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_createAppUploadFetchBookAppFetchApp" time="24.616" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_bookAppointmentForExpiredApplication" time="15.890" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <failure type="java.lang.AssertionError" message="Failed stabilized connection with preregdb expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed stabilized connection with preregdb expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at io.mosip.preregistration.dao.PreregistrationDAO.setDate(PreregistrationDAO.java:65)
	at io.mosip.preregistration.tests.IntegrationScenarios.bookAppointmentForExpiredApplication(IntegrationScenarios.java:578)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </failure>
  </testcase> <!-- preReg_IntTst_bookAppointmentForExpiredApplication -->
  <testcase name="preReg_IntTst_cancelAppointmentForExpiredApplication" time="23.737" classname="io.mosip.preregistration.tests.IntegrationScenarios">
    <failure type="java.lang.AssertionError" message="Failed stabilized connection with preregdb expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed stabilized connection with preregdb expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at io.mosip.preregistration.dao.PreregistrationDAO.setDate(PreregistrationDAO.java:65)
	at io.mosip.preregistration.tests.IntegrationScenarios.cancelAppointmentForExpiredApplication(IntegrationScenarios.java:497)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:42)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:30)
]]>
    </failure>
  </testcase> <!-- preReg_IntTst_cancelAppointmentForExpiredApplication -->
  <testcase name="preReg_IntTst_createMultipleAppDeleteFewFetchAllAppsByUserId" time="33.950" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_discardBookedAppointment" time="34.618" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_retrivePreRegistrationDataAfterUploadingDocument" time="9.036" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
  <testcase name="preReg_IntTst_getStatusOfCanceledAppointment" time="27.203" classname="io.mosip.preregistration.tests.IntegrationScenarios"/>
</testsuite> <!-- io.mosip.preregistration.tests.IntegrationScenarios -->
